### DISCUSSION TITLE
duplex off problem

### DISCUSSION THREAD

**Discussion by f4axk** [egzumer/uv-k5-firmware-custom#397] [General]
duplex off problem

when duplex off goes to tx on  -0.180000

egzzumer 0.21

**Comment by dadanielat** [comment]
I also noticed that, an offset is displayed if duplex is set to off, but this seems to be only a cosmetic problem.

**Reply by egzumer** [reply]
Where did you get -0.180000 from?
Duplex off sets a shift in a radio to TX at 0MHz which results in TX DISABLE message.

**Reply by f4axk** [reply]
egzumer 0.21
chirp_egzumer 0.8.1

![off](https://github.com/egzumer/uv-k5-firmware-custom/assets/68162981/bf2cf4bb-eaa9-4434-b8e0-919a455fcfb6)

**Comment by f4axk** [comment]
Hello egzumer
Have you solved the problem on duplex off?
which displays 0.180000 and goes into transmission see the photos on the previous post.
Thanks 73s

**Comment by radonslide** [comment]
Same effect happens on my UV-K5 with CB frequencies set to duplex off.
Firmware build from the current available source .

73 Mike

**Comment by egzumer** [comment]
I have no idea how you do that, I checked both CB and 145.775, no issue whatsoever. I get TX DISABLE.
Go to menu 8 and see what is the offset, it should be equal to channel frequency.

**Reply by f4axk** [reply]
I just checked, this only happens when Unlok All mode is enabled

**Reply by radonslide** [reply]
Thank you for your research. 
I only looked at the frequencies yesterday, but they were clearly correct and equal.

**Comment by egzumer** [comment]
Fixed https://github.com/egzumer/uv-k5-firmware-custom/commit/dff4384dbe74df72aa1413592f28fea8336a7305
